### ISSUE TITLE
All `s` are replaced by `\\s` in the generated .desktop

### ISSUE THREAD

**Issue by jordanamr** [SpacingBat3/ReForged#2]
Using this maker config :
```
{
    "name": "@reforged/maker-appimage",
    "platforms": [
        "linux"
    ],
    "config": {
        "options": {
            "name": "arena-returns-launcher",
            "icon": "src/assets/icon/icon.png",
            "categories": [
                "Game",
                "StrategyGame"
            ]
        }
    }
}
```

All "s" in the generated .desktop are replaced by "\\\s". I saw that the appimage wouldn't start, so I extracted it and looked at the .desktop and saw this:
```
[Desktop Entry]
Type=Application
Name=Arena Return\\s Launcher
Exec=arena-return\\s-launcher
Icon=arena-return\\s-launcher
Categorie\\s=Game;StrategyGame;
X-AppImage-Name=arena-return\\s-launcher
X-AppImage-Ver\\sion=1.0.5-SNAPSHOT
X-AppImage-Arch=x86_64
```

This is probably caused by https://github.com/SpacingBat3/ReForged/blob/6f8fa6ecd71ff9943183fd8a6c797b56f6b44178/makers/appimage/src/utils.ts#L105

**Comment by jordanamr** [comment]
Hello !
I've updated to `"@reforged/maker-appimage": "^2.1.2"` but I still encounter the same issue :
![image](https://user-images.githubusercontent.com/55905781/209416999-9804ffd1-cfa7-47c6-9e36-f167d5594143.png)

Am I missing something ?

EDIT: After inspecting my node_modules, the package.json of maker-appimage is indeed at 2.1.2, but the problematic code can still be found in dist :
![image](https://user-images.githubusercontent.com/55905781/209417067-01d921a0-7825-4c4b-8b5f-a22c6a346804.png)


EDIT 2: Seems like it only affects 2.1.2, the 2.1.1 version with unrelated changes was correctly published (even tho it won't build because of the unrelated changes)

**Comment by SpacingBat3** [comment]
@jordanamr I've published 2.1.3, this should work as I made sure now I built it with clean workdir in `git`. It also contains some more changes reverted I didn't want to make into `2.x.y` (I'm in process to release next major release).

**Comment by jordanamr** [comment]
@SpacingBat3 Thank you :)
